directives/run-async.js

```diff
@@ -100,7 +102,8 @@ export const runAsync = directive((key, task, templates) => (part) => {
 			await 0;
 			const currentRunState = runs.get(part);
 			if (currentRunState === runState && currentRunState.state === 'pending') {
-				part.startNode.parentNode.dispatchEvent(new CustomEvent('pending-state', {
+				const element = isElement(part.startNode.parentNode) ? part.startNode.parentNode : part.startNode.parentNode.host;
```

Comment:
Is it possible that we can just use the standard `nodeType` property rather than inlcuding `lodash`?

Reply:
I was thinking the same yeah. Here's what [lodash is doing](https://github.com/lodash/lodash/blob/master/isElement.js)... hoping we can just check `part.startNode.parentNode.nodeType === 1`.

Reply:
Is it sufficient? Lodash is checking a bunch more, but maybe it's not necessary?

Reply:
I think so yes.

Reply:
ok, I'll update it. I guess we already know it's a node.

Reply:
I'd say start with what the browser can do natively, and if need be then we can look at importing  3rd party lib or rolling our own helper.